modules/Data.py

```diff
@@ -106,3 +106,12 @@ def get_lending_currencies():
     for cur in lending_balances:
         currencies.append(cur)
     return currencies
+
+
+def truncate(f, n):
```

Comment:
Okay I will add that as a comment, wait for approval of laxdog's PRs, then merge this.